content/hardware/02.hero/boards/uno-r4-wifi/tutorials/stack-trace/stack-trace.md

```diff
@@ -1,14 +1,22 @@
 ---
-title: Debug runtime errors on UNO R4 WiFi using stack trace
+title: UNO R4 WiFi Stack Trace Debug Runtime Errors
+description: Learn how to get information from the Cortex-M4 processor on a runtime error.
+author: Sebastian Wikström
+tags: [Debug, Runtime Errors, Stack Trace]
 ---
 
-The UNO R4 WiFi uses [CmBacktrace](https://github.com/armink/CmBacktrace) to print useful information from the **Arm Cortex-M4** processor as serial output when a runtime error occurs. The output includes an `addr2line` command that can used to produce a **stack trace** (also **stack backtrace**, or **stack traceback**), which can be used to find the source of the error.
+The Arduino UNO R4 WiFi uses [CmBacktrace](https://github.com/armink/CmBacktrace) to print useful information from the **Arm Cortex-M4** processor as serial output when a runtime error occurs. The output includes an `addr2line` command that can used to produce a **stack trace** (also **stack backtrace**, or **stack traceback**), which can be used to find the source of the error.
 
----
+In this tutorial, we will show to enable this feature.
 
-## Get runtime error information in the Serial Monitor
+## Hardware & Software Needed
 
-Follow these steps:
+- [Arduino IDE](https://www.arduino.cc/en/software)
+- [Arduino UNO R4 WiFi](https://store.arduino.cc/products/uno-r4-wifi)
+
+## Get Runtime Error
```

Comment:
```suggestion
## View the Runtime Error
```
To me, "Get Runtime Error" makes it sound like the goal is to make the error occur in the first place, rather than viewing an error that has already occured.